libraries/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -276,7 +279,9 @@ export class OperationExecutionManager {
         } else {
           await record.executeAsync({
             onStart: onOperationStartAsync,
-            onResult: onOperationCompleteAsync
+            beforeResult: beforeOperationResult,
+            onResult: async (finishedRecord: OperationExecutionRecord) =>
+              this._onOperationComplete(finishedRecord)
```

Comment:
```suggestion
            onResult: this._onOperationComplete
```

Reply:
I updated `this._onOperationComplete` to be async to match the types here.

Reply:
You can un-async `_onOperationComplete` if you change the expected return type of `onResult` to `Promise<void> | void`